vaadin-checkbox-flow-integration-tests/src/main/java/com/vaadin/flow/component/checkbox/tests/CheckboxGroupDemoPage.java

```diff
@@ -54,6 +60,27 @@ protected void initView() {
         addDisabledItems();
         addReadOnlyGroup();
         addComponentWithThemeVariant();
+        addDataViewDemo();
+    }
+
+    private void addDataViewDemo() {
+        CheckboxGroup<String> checkboxGroup = new CheckboxGroup<>();
+
+        final CheckboxGroupListDataView<String> dataView =
+                checkboxGroup.setDataProvider(DataProvider.ofItems("Item1", "Item2", "Item3"));
+
+        Button filterButton = new Button("Filter",
+                event -> dataView.withFilter(item -> item.equals("Item2")));
+
+        Button orderButton = new Button("Order",
+                event -> dataView.withOrder(String::compareTo));
+
+        String nextItem = dataView.getNextItem("Item2");
+        String previousItem = dataView.getPreviousItem("Item2");
+        boolean hasNextItem = dataView.hasNextItem("Item3");
+        boolean hasPreviousItem = dataView.hasPreviousItem("Item3");
```

Comment:
These are un-used and should be removed or the demo expanded.

Reply:
This is actually some old pieces of code, which are still there erroneously and have to be removed.

Reply:
Removed.